projects/cm_mcu/MonitorI2CTask.c

```diff
@@ -0,0 +1,195 @@
+/*
+ * MonitorI2CTask.c
+ *
+ *  Created on: June 30, 2022
+ *      Author: pkotamnives
+ *      Monitor temperatures, and statuses of firefly ports via I2C BUS
+ * pass in addresses via parameter to the task.
+ */
+
+// includes for types
+#include <stdint.h>
+#include <stdbool.h>
+#include <stdio.h>
+#include <string.h>
+
+// memory mappings
+#include "inc/hw_types.h"
+#include "inc/hw_memmap.h"
+
+// FreeRTOS
+#include "FreeRTOS.h"
+#include "FreeRTOSConfig.h"
+#include "task.h"
+
+// local includes
+#include "common/i2c_reg.h"
+#include "common/utils.h"
+#include "common/smbus_helper.h"
+#include "common/smbus_units.h"
+#include "MonitorI2CTask.h"
+#include "common/power_ctl.h"
+#include "common/log.h"
+#include "Tasks.h"
+#include "I2CCommunication.h"
+
+// local prototype
+
+void Print(const char *str);
+
+// break out of loop, releasing semaphore if we have it
+
+#define release_break()           \
+  {                               \
+    xSemaphoreGive(args->xSem);   \
+    break;                        \
+  }
+
+// get a mask of a register address given a starting bit field (shift) and the width
+
+void make_bitmask(unsigned int width, unsigned int shift, uint8_t *mask)
```

Comment:
I think this function is no longer used, right?

Reply:
yes, just remove it. Done.